CHANGES_NEXT_RELEASE

```diff
@@ -1,8 +1,10 @@
-- Add: support for MongoDB 2.6 (MongoDB 2.4 should *not* be used any longer as inverted geo-queries will not work in that version) (Issue #415)
+- Add: support for MongoDB 2.6/3.0 (MongoDB 2.4 should *not* be used any longer as inverted geo-queries will not work in that version) (Issue #415)
```

Comment:
Having checked that also work with 3.0 (from a functional point of view), let's adjust the entry.

NTC (just informative)